### ISSUE TITLE
FFT/OpenCL tests require user input to select the device

### ISSUE THREAD

**Issue by kif** [silx-kit/silx#2383]
this locks the build ...

**Comment by kif** [comment]
```
test_numpy_fft (silx.math.fft.test.test_fft.TestNumpyFFT) ... ok
test_numpy_fft (silx.math.fft.test.test_fft.TestNumpyFFT) ... ok
Choose platform:
[0] <pyopencl.Platform 'Intel Gen OCL Driver' at 0x7fed50181540>
[1] <pyopencl.Platform 'Intel(R) OpenCL' at 0x30d7490>
[2] <pyopencl.Platform 'Portable Computing Language' at 0x7fed4c975020>
```

**Comment by pierrepaleo** [comment]
Should be fixed by #2384.
